trade-finance-manager-ui/server/controllers/utilisation-reports/helpers/edit-payment-helper.test.ts

```diff
@@ -3,11 +3,23 @@ import { Currency, CurrencyAndAmount } from '@ukef/dtfs2-common';
 import { getEditPaymentViewModel, getEditPaymentViewModelWithFormValues } from './edit-payment-helper';
 import { aPaymentDetailsWithFeeRecordsResponseBody, aPayment, aFeeRecord } from '../../../../test-helpers';
 import { GetPaymentDetailsWithFeeRecordsResponseBody } from '../../../api-response-types';
+import { RECONCILIATION_FOR_REPORT_TABS } from '../../../constants/reconciliation-for-report-tabs';
 import { PaymentErrorsViewModel, SortedAndFormattedCurrencyAndAmount } from '../../../types/view-models';
 import { EditPaymentFormValues } from '../../../types/edit-payment-form-values';
 import { EMPTY_PAYMENT_ERRORS_VIEW_MODEL } from './payment-form-helpers';
+import { getReconciliationForReportHref } from './get-reconciliation-for-report-href';
+
+jest.mock('./get-reconciliation-for-report-href');
 
 describe('edit-payment-helper', () => {
+  beforeEach(() => {
+    jest.mocked(getReconciliationForReportHref).mockReturnValue('/utilisation-reports/1');
```

Comment:
Sorry for not spotting before, do we have to mock this? As it's just returning a URL, I think should be consumed in the unit test(s) rather than mocking it? e.g

```ts
const expected = getReconciliationForReportHref(reportId, redirectTab);

expect(viewModel.backLinkHref).toEqual(expected);
```

Reply:
Updated this as it is returning something fairly simple as you say, thanks